client/src/components/LayoutComponents/Dashboard/index.jsx

```diff
@@ -18,6 +19,7 @@ class Dashboard extends Component {
 	render() {
 		return (
 			<Fragment>
+				<NavBar history={this.props.history} />
 				<h1>Welcome to your Dashborad {this.state.user.displayName}</h1>
```

Comment:
Dashboard

Reply:
? wt?

Reply:
@karmelyoei typo, Dashboard not Dashborad

Reply:
it doesn't matter it will be changed for another branch frontend